Library/Homebrew/formula.rb

```diff
@@ -1781,8 +1781,11 @@ def generate_completions_from_executable(*commands,
       popen_read_args << shell_parameter if shell_parameter.present?
       popen_read_args.flatten!
 
+      popen_read_options = {}
+      popen_read_options[:err] = :err unless ENV["HOMEBREW_STDERR"]
```

Comment:
I guess an alternative would be setting `HOMEBREW_STDERR` in CI. Would need to know full impact.